user/g4music/template.py

```diff
@@ -0,0 +1,29 @@
+pkgname = "g4music"
+pkgver = "3.8.2"
+pkgrel = 0
+build_style = "meson"
+# Only checks the validity of the desktop files
+configure_args = ["-Dunit_tests=disabled"]
```

Comment:
but they run instantly and take zero time, so it's fine to leave them enabled and not touch it